packages/react-dom/src/__tests__/ReactTestUtils-test.js

```diff
@@ -53,7 +53,12 @@ describe('ReactTestUtils', () => {
     MockedComponent.prototype.render = jest.fn();
 
     // Patch it up so it returns its children.
-    ReactTestUtils.mockComponent(MockedComponent);
+    expect(() =>
+      ReactTestUtils.mockComponent(MockedComponent),
+    ).toLowPriorityWarnDev(
+      'ReactTestUtils.mockComponent() is deprecated. ' +
+        'Use shallow rendering or jest.mock() instead.',
```

Comment:
Let's add an fb.me link that either points to a gist or a warning page with a drop-in replacement code? People who rely on it might not understand what mockComponent was doing exactly.